docs/UserGuide.md

```diff
@@ -252,7 +252,7 @@ Format: `view STUDENT_ID`
 * The `STUDENT_ID` starts with A, ends with a letter, and it must be 9 characters long.
```

Comment:
I think it would be good to add a line saying that this feature is not applicable to the archived list.

Reply:
I've added the disclaimer.